content/pt-br/docs/concepts/architecture/cloud-controller.md

```diff
@@ -1,162 +1,115 @@
+### Service Controller
 
-Para obter mais informações sobre o desenvolvimento de plug-ins, consulte[Desenvolvendo o Cloud Controller Manager](/docs/tasks/administer-cluster/developing-cloud-controller-manager/).
+Os {{< glossary_tooltip text="Services" term_id="service" >}} integram-se com componentes da infraestrutura de nuvem, como balanceadores de carga gerenciados, endereços IP, filtragem de pacotes de rede e verificação de integridade dos alvos. O service controller interage com as APIs do seu provedor de nuvem para configurar balanceadores de carga e outros componentes de infraestrutura quando você declara um recurso de Service que os requer.
 
 ## Autorização
 
-Esta seção divide o acesso necessário em vários objetos da API pelo CCM para executar suas operações.
-
-### Node Controller
+Esta seção detalha o acesso que o cloud controller manager precisa em vários objetos da API para realizar suas operações.
 
-O Node Controller só funciona com objetos Node. Ele requer acesso total para obter, listar, criar, atualizar, corrigir, assistir e excluir objetos Node.
+### Node Controller 
 
-v1/Node:
+O Node controller trabalha apenas com objetos Node. Ele requer acesso completo para ler e modificar objetos Node.
 
-- Get
-- List
-- Create
-- Update
-- Patch
-- Watch
-- Delete
+`v1/Node`:
 
-### Rote Controller
+- get
+- list
+- create
+- update
+- patch
+- watch
+- delete
 
-O Rote Controller escuta a criação do objeto Node e configura as rotas apropriadamente. Isso requer acesso a objetos Node.
+### Route Controller
+O controlador de roteadores monitora a criação de objetos Node e configura rotas de forma apropriada. Ele requer acesso de leitura (Get) aos objetos Node.
 
-v1/Node:
+`v1/Node`:
 
-- Get
+- get
 
 ### Service Controller
 
-O Service Controller escuta eventos de criação, atualização e exclusão de objeto de serviço e, em seguida, configura pontos de extremidade para esses serviços de forma apropriada.
+O service controller monitora eventos de criação(**create**), atualização(**update**) e exclusão(**delete**) de objetos do tipo Service e, em seguida, configura Endpoints para esses serviços de forma apropriada (para EndpointSlices, o kube-controller-manager gerencia esses sob demanda).
 
-Para acessar os Serviços, é necessário listar e monitorar o acesso. Para atualizar os Serviços, ele requer patch e atualização de acesso.
+Para acessar a API de Services, ele requer acesso para listar (**list**) e monitorar (**watch**). Para atualizar Services, ele precisa de acesso para aplicar patch (patch) e atualizar (update).
 
-Para configurar endpoints para os Serviços, é necessário acesso para criar, listar, obter, assistir e atualizar.
+Para configurar o recurso de `Endpoints` para os Services, ele necessita de acesso para criar (**create**), listar (**list**), obter (**get**), monitorar (**watch**) e atualizar (**update**).
```

Comment:
Como Endpoint é um objeto do Kubernetes, o ideal é que não seja formatado como código, mas apenas com letra maiúscula conforme o guia de estilo: https://kubernetes.io/pt-br/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects

```suggestion
Para configurar o recurso de Endpoints para os Services, ele necessita de acesso para criar (**create**), listar (**list**), obter (**get**), monitorar (**watch**) e atualizar (**update**).
```